pkg/cmd/admin/admin.go

```diff
@@ -45,54 +47,55 @@ func NewCommandAdmin(name, fullName string, out io.Writer, errout io.Writer) *co
 
 	groups := templates.CommandGroups{
 		{
-			Message: "Basic Commands:",
+			Message: "Component Installation:",
+			Commands: []*cobra.Command{
+				router.NewCmdRouter(f, fullName, "router", out),
+				exipfailover.NewCmdIPFailoverConfig(f, fullName, "ipfailover", out, errout),
+				registry.NewCmdRegistry(f, fullName, "registry", out),
+			},
+		},
+		{
+			Message: "Security and Policy:",
 			Commands: []*cobra.Command{
 				project.NewCmdNewProject(project.NewProjectRecommendedName, fullName+" "+project.NewProjectRecommendedName, f, out),
 				policy.NewCmdPolicy(policy.PolicyRecommendedName, fullName+" "+policy.PolicyRecommendedName, f, out, errout),
 				groups.NewCmdGroups(groups.GroupsRecommendedName, fullName+" "+groups.GroupsRecommendedName, f, out),
+				cert.NewCmdCert(cert.CertRecommendedName, fullName+" "+cert.CertRecommendedName, out, errout),
+				admin.NewCommandOverwriteBootstrapPolicy(admin.OverwriteBootstrapPolicyCommandName, fullName+" "+admin.OverwriteBootstrapPolicyCommandName, fullName+" "+admin.CreateBootstrapPolicyFileCommand, out),
 			},
 		},
 		{
-			Message: "Install Commands:",
+			Message: "Node Management:",
 			Commands: []*cobra.Command{
-				router.NewCmdRouter(f, fullName, "router", out),
-				exipfailover.NewCmdIPFailoverConfig(f, fullName, "ipfailover", out, errout),
-				registry.NewCmdRegistry(f, fullName, "registry", out),
+				admin.NewCommandNodeConfig(admin.NodeConfigCommandName, fullName+" "+admin.NodeConfigCommandName, out),
+				node.NewCommandManageNode(f, node.ManageNodeCommandName, fullName+" "+node.ManageNodeCommandName, out, errout),
+				cmdutil.ReplaceCommandName("kubectl", fullName, kubectl.NewCmdCordon(f.Factory, out)),
```

Comment:
I expected these as `oadm node cordon` as opposed to `oadm cordon`.  Seems like these are directly related to nodes.